src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -26,9 +27,23 @@ class CalculatorForm extends React.Component {
     this.renderWorking = this.renderWorking.bind(this);
   }
 
-  handleInputChange(event) {
-    const { name: field, value } = event.target;
-    this.props.onInputChange({ field, value });
+  getExtensions() {
+    const { profile } = this.props;
+    const facilityMap = profile.attributes.facilityMap;
+    const profileFacilityCode = profile.attributes.facilityCode;
+    let extensions;
+    if (profileFacilityCode === facilityMap.main.institution.facilityCode) {
+      extensions = profile.attributes.facilityMap.main.extensions;
+    } else {
+      facilityMap.main.branches.forEach(branch => {
```

Comment:
I presume you'll want to stop once a matching branch is found, but rather than `break`, I would suggest the following:

```
const matchedBranch = facilityMap.main.branches.find(branch =>
  branch.institution.facilityCode === profile.attributes.facilityCode
);
({ extensions } = matchedBranch);
```